test/tlsconnection.js

```diff
@@ -780,10 +888,20 @@ describe('the ClientConnection class', () => {
         });
       });
 
+      it('accepts and ignores a NewSessionTicket message', async () => {
+        const encryptor = await testHelpers.makeEncryptionState(client._keyschedule.serverApplicationTrafficSecret, 0);
+        const newSessionTicket = new NewSessionTicket(0, 0, zeros(0), zeros(0), []);
```

Comment:
It might pay to feed it a bigger ticket than that.  I'd suggest at least one extension, but none are allowed.  Maybe test that an invalid NST is rejected.